scripts/api/data/licenses/licenseEUPL-1.2.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "EUPL-1.2",
+  "uri": "https://joinup.ec.europa.eu/page/eupl-text-11-12",
```

Comment:
This does seem to be the right uri. The DataCite Fabrica code seems to pick the first "seeAlso". Here is their data: https://github.com/datacite/bracco/blob/df4ac2030774324adeaa72e2978c29dfb463ee3d/app/spdx.js#L1470

It shows five "seeAlso" URLs:

```
      seeAlso: [
        'https://joinup.ec.europa.eu/page/eupl-text-11-12',
        'https://joinup.ec.europa.eu/sites/default/files/custom-page/attachment/eupl_v1.2_en.pdf',
        'https://joinup.ec.europa.eu/sites/default/files/inline-files/EUPL%20v1_2%20EN(1).txt',
        'http://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32017D0863',
        'https://opensource.org/licenses/EUPL-1.1'
      ],
```